compiler/compiler-core.h

```diff
@@ -1,5 +1,5 @@
 // Compiler for PHP (aka KPHP)
-// Copyright (c) 2020 LLC «V Kontakte»
+// Copyright (c) 2021 LLC «V Kontakte»
```

Comment:
Let's not change any file headers (this comment is actual for all files below). It's okay, that the year 2020 remains here.